### PR TITLE
DVX-4446: Adds IsHostname checker to trek

### PR DESCRIPTION
**Pull request by ifosch**

### CODE REVIEW

File: mcc/trek/trek/trek.go [hunk 1]
```diff
@@ -6,8 +6,21 @@ import (
 	"io"
 	"log"
 	"os"
+	"regexp"
 )
 
+// RFC1123HostnameRegExp : Hostname Regular expression according with RFC1123HostnameRegExp.
+const RFC1123HostnameRegExp = "^(([a-zA-Z0-9]|[a-zA-Z0-9][a-zA-Z0-9-]*[a-zA-Z0-9]|\\*)\\.)*([A-Za-z0-9]|[A-Za-z0-9][A-Za-z0-9-]*[A-Za-z0-9])$"
+
+// IsHostname returns true is argument can be considered a hostname.
```
Comment (miguelbernadi) [inline mcc/trek/trek/trek.go]: ...returns true _if_ argument...

**Comment by miguelbernadi** [comment]
+1

File: mcc/trek/trek/trek.go (hunk 1 above)
Comment (ifosch) [reply mcc/trek/trek/trek.go]: True!

**Comment by EdLittle** [comment]
+1

**Pull request merged**